### ISSUE TITLE
Sign up issue

### ISSUE THREAD

**Issue by abdulrahim3301** [the-urlist/home#12]
Not sure how to explain that in the title and not sure if I should mark it as an issue here on GitHub but I just tried URList and to test it out, I didn't login with Twitter initially and made a link that I normally use over the internet and once I realized I missed a link to add in the master link I signed up through Twitter BUT now it's a new account and it isn't letting me use that link again.. :(
So basically I made that link myself and now it's..."Taken by someone else".
Please help out.
